webrtc.html

```diff
@@ -9849,6 +9847,31 @@ <h2>Methods</h2>
         </section>
       </div>
     </section>
+    <section>
+      <h3>canInsertDTMF algorithm</h3>
+      <p>To <dfn>determine if DTMF can be sent</dfn> for an <code><a>RTCDTMFSender</a></code>
+      instance <var>dtmfSender</var>, the user agent MUST queue a task that runs the following steps:</p>
+      <ol>
+        <li>Let <var>sender</var> be the <code><a>RTCRtpSender</a></code> associated with
+        <var>dtmfSender</var>.</li>
+        <li>Let <var>transceiver</var> be the <code><a>RTCRtpTransceiver</a></code>
+        associated with <var>sender</var>.</li>
+        <li>Let <var>connection</var> be the <code><a>RTCPeerConnection</a></code> associated with
+        <var>transceiver</var>.</li>
+        <li>If <var>connection</var>'s <code><a>RTCPeerConnectionState</a></code> is not <code>"connected"</code>
+        return <code>false</code> and abort these steps.</li>
+        <li>If <var>sender</var>'s <a>[[\SenderTrack]]</a> is <code>null</code>
+        return <code>false</code> and abort these steps.</li>
+        <li>If <var>transceiver</var>'s <a>[[\CurrentDirection]]</a> is neither <code>"sendrecv"</code>
+        nor <code>"sendonly"</code> return <code>false</code> and abort these steps.</li>
+        <li>If <var>sender</var>'s <a>[[\SendEncodings]]</a><code>[0].active</code> is
+        <code>false</code> return <code>false</code> and abort these steps.</li>
+        <li>If, for <var>sender</var>, no codec (which the user agent is currently capable
+        of sending) with mimetype <code>"audio/telephone-event"</code> has been
+        negotiated for sending, return <code>false</code> and abort these steps.</li>
+        <li>Return <code>true</code>.</li>
```

Comment:
If  we said *"Otherwise, return <code>true</code>."* here, could we remove *"and abort these steps"* from all preceeding steps?

Reply:
Fixed in https://github.com/w3c/webrtc-pc/pull/1980/commits/27fd6925776df0d655589bbf8ab06110e6763f88.